src/components/Table/HeaderCell/index.tsx

```diff
@@ -92,6 +96,12 @@ function HeaderCell(props: HeaderCellProps) {
             <div className={_cs(titleClassName, styles.title)}>
                 {title}
             </div>
+            {infoTitle && infoDescription && (
```

Comment:
Do we need both? Or, is this either title or description?

Reply:
We need both!